ui/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.17",
   "dependencies": {
     "@babel/runtime": "^7.9.2",
-    "@inveniosoftware/react-invenio-app-ils": "^1.0.0-alpha.48",
+    "@inveniosoftware/react-invenio-app-ils": "^1.0.0-alpha.51",
```

Comment:
when you are upgrading a package, you have to re-create package-lock.json file as well and commit it as well